pyproject.toml

```diff
@@ -17,7 +17,7 @@ test-command = [
 [tool.cibuildwheel.macos.environment]
 BOOST_BUILD_PATH = "/tmp/boost/tools/build"
 BOOST_ROOT = "/tmp/boost"
-BOOST_VERSION = "1.80.0"
+BOOST_VERSION = "1.81.0"
```

Comment:
what does this affect?

Reply:
Unsure what you mean? This is for the macos wheels

Reply:
I'm not very familiar with how the wheels are build, so I'm curious what logic looks at this constant and installs the corresponding version of boost.

Reply:
I'm not a fan of toml... Check the previous table for the macos boost install, this table is nested env vars for it